src/app/[locale]/(home)/orders/page.tsx

```
@@ -0,0 +1,10 @@
+import React from 'react'
+
```

ESLint

[error] Missing semicolon.

(semi)